src/agent.js

```diff
@@ -43,6 +43,8 @@ const Auth = {
     requests.post("/users/register", { user: { email, password } })
 };
 
+console.log("auth current: " + Auth.current());
```

Comment:
For security purposes I would be careful about what data of `Auth` object gets saved into server logs